test/unit/auth.js

```diff
@@ -62,58 +62,68 @@ describe('DropboxAuth', () => {
 
     it('throws an error if the redirect url isn\'t set and type is code', () => {
       const dbx = new Dropbox({ clientId: 'CLIENT_ID' });
-      chai.assert.equal(
+      return chai.expect(
```

Comment:
I'm sure its fine but why do we return this?

Reply:
When using chai-as-promised, you have to return the promise fulfillment or the test will just pass even if it's incorrect.